src/client.rs

```diff
@@ -740,11 +719,11 @@ impl Spotify {
         position: Option<i32>,
     ) -> ClientResult<PlaylistResult> {
         let uris = track_ids.into_iter().map(|id| id.uri()).collect::<Vec<_>>();
+        let params = build_json! {
+            "uris": uris,
+            "position": position,
```

Comment:
The `position` should be optional, right?

Reply:
Fixed